src/core/forms/forms.py

```diff
@@ -106,8 +106,8 @@ class Meta:
 
 class PasswordResetForm(forms.Form):
 
-    password_1 = forms.CharField(widget=forms.PasswordInput, label=_('Password 1'))
-    password_2 = forms.CharField(widget=forms.PasswordInput, label=_('Password 2'))
+    password_1 = forms.CharField(widget=forms.PasswordInput, label=_('Password'))
+    password_2 = forms.CharField(widget=forms.PasswordInput, label=_('Confirm Password'))
```

Comment:
Can we also update "Password 2" entries on .po files to avoid reverting this translation on live installs?